tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,6 +125,14 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_revert_commit(self):
+        self.git_commit('Revert : identifies revert commit\n')
```

Comment:
Actually, you can maybe try to invoke `git revert` itself. That way we can see if we are really compliant with git's revert commits. So commit some arbitrary stuff, and then revert and test.